src/meson.build

```diff
@@ -222,6 +222,7 @@ mangohud_opengl_shared_lib = shared_library(
   opengl_files,
   vklayer_files,
   util_files,
+  gnu_symbol_visibility : 'hidden',
```

Comment:
Not sure if you or I added this, but it seems irrelevant to this PR?

Reply:
it was part of your original code, I thought it was maybe used with the DEEPBIND for something